src/actions/guides/database/querying.cr

```diff
@@ -469,7 +485,7 @@ class Guides::Database::Querying < GuideAction
       def recently_completed_admin_tasks
         admin(true)
           .join_tasks
```

Comment:
This  can  be removed now. May be worth  mentioning an inner join will already be added. And showing an example where you can do some other join  with the `auto_inner_join` set to false